build.gradle

```diff
@@ -20,7 +20,7 @@ dependencies {
     implementation 'com.squareup.okhttp3:okhttp:4.3.1'
     implementation 'com.github.weisj:darklaf-core:2.2.3'
     implementation 'com.github.weisj:darklaf-theme:2.2.3'
-    implementation 'com.formdev:svgSalamander:1.1.2.1'
```

Comment:
@yaute74 You may need to check whether this will affect the changes you are working on.

Reply:
Ok. I will first work on issues before come back to the svg, we're starting to get a lot of issue :(